next/next.config.js

```diff
@@ -1,10 +1,24 @@
 const { i18n } = require("./next-i18next.config");
+var crypto = require("crypto");
 
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
   // Only generate standalone output in circle ci:
   output: process.env.CIRCLECI ? "standalone" : undefined,
+  generateBuildId: async () => {
+    // This environment variable is set by CircleCI.
+    // adjust this to your needs if you use another CI/CD tool.
+    return process.env.CIRCLE_BUILD_NUM
+      ? `build-id-${process.env.CIRCLE_BUILD_NUM}`
+      : // If no build number is available, we generate a random build ID.
+        crypto.randomBytes(20).toString("hex");
+  },
+  cacheHandler:
+    // If the environment has a redis host set, we use a cache handler:
+    process.env.NODE_ENV === "production"
+      ? require.resolve("./cache-handler.mjs")
+      : undefined,
   images: {
     remotePatterns: [
       {
```

Comment:
@joshua-scott  this is the feature that generates a random build it (I was wrong thinking that we had merged it already)